### PR TITLE
Exercise lists now update on user selection showing only exercises for each body part

### PR DESCRIPTION
**Pull request by Darango84**

### CODE REVIEW

File: src/PageView/Elements/Elements.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ export default function Elements() {
                 <h1 className="separator-right">Left Separator</h1>
             </div>
             <div className="grid">
-                <div>
+                {/* <div>
```
Comment (phapsides) [inline src/PageView/Elements/Elements.js:27]: Is the reason you commenting this out the warnings you get in the console?

File: src/PageView/WorkoutsMain/WorkoutExerciseList/ExerciseListDetails.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                }  
+                </div>                    
+            </div>
+        )
+    }
+}
+
+
+
+// export default function WorkoutList() {
```
Comment (phapsides) [inline src/PageView/WorkoutsMain/WorkoutExerciseList/ExerciseListDetails.js:49]: Delete if not needed

File: src/PageView/Elements/Elements.js (hunk 1 above)
Comment (Darango84) [reply src/PageView/Elements/Elements.js:27]: yes that is why

File: src/PageView/WorkoutsMain/WorkoutExerciseList/ExerciseListDetails.js (hunk 2 above)
Comment (Darango84) [reply src/PageView/WorkoutsMain/WorkoutExerciseList/ExerciseListDetails.js:49]: thanks, done

File: src/PageView/Elements/Elements.js (hunk 1 above)
Comment (phapsides) [reply src/PageView/Elements/Elements.js:27]: Don't think is relevant with what you are trying to address here and they are only warnings.

If you would like to address this here I would suggest you replace '#' with '/'

**Pull request merged**